dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -653,6 +657,49 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
         return limits;
     }
 
+    @POST
+    @Path("/{userId}/services/refresh")
```

Comment:
Path should maybe be "/{userId}/services/sync" to distinguish the light operation of looking for organizations and repo names as opposed to the "heavy" operation of actually pulling down versions and all the files for those versions which we can continue calling refresh.

Reply:
Changed to sync